guides/v2.1/cloud/project/project-routes-more-redir.md

```diff
@@ -34,114 +35,73 @@ http://{default}/:
-
-Optional, defaults to `false`. Specifies whether the path key should be interpreted asa PCRE regular expression.
-
-{% collapsible Click to show/hide content %}
-
-In the following example, a request to `http://example.com/regexp/a/b/c/match` redirects to `http://example.com/a/b/c`:
-
-```yaml
-http://{default}/:
-    type: upstream
-    redirects:
-        paths:
-            "/regexp/(.*)/match":
-                to: "http://example.com/$1"
-                regexp: true
-```
-
-{% endcollapsible %}
-
-### `prefix` {#cloud-route-partial-prefix}
-Specifies whether or not to redirect both the path and all its children or just the path itself. Defaults to `true`, but is not supported if `regexp` is `true`.
-
-{% collapsible Click to show/hide content %}
-
-For example,
-
-```yaml
-http://{default}/:
-    type: upstream
-    redirects:
-        paths:
-            "/from":
-                to: "http://{default}/to"
-                prefix: true
-```
-
-In the preceding example, if `prefix` is set to `true`, `/from` redirects to `/to` and `/from/another/path` will redirect to `/to/another/path`
-
-If `prefix` is set to `false`, `/from` triggers a redirect, but `/from/another/path` does not.
-
-{% endcollapsible %}
-
-### `append_suffix` {#cloud-route-partial-append}
-Determines if the suffix is carried over with the redirect. Defaults to `true`, but not supported if `regexp` is `true` *or* if `prefix` is `false`.
-
-{% collapsible Click to show/hide content %}
-
-For example,
-
-```yaml
-http://{default}/:
-    type: upstream
-    redirects:
-        paths:
-            "/from":
-                to: "http://{default}/to"
-                append_suffix: false
-```
-
-The preceding example results in `/from/path/suffix` redirecting to just `/to`.
-
-If `append_suffix` is set to its default value of `true`, `/from/path/suffix` redirects to `/to/path/suffix`.
-
-{% endcollapsible %}
-
-### `code` {#cloud-route-partial-code}
-
-Specifies the HTTP status code. Valid status codes are [`301` (Moved Permanently)](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.2), [`302`](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.3), [`307`](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.8), and [`308`](https://tools.ietf.org/html/rfc7238). Defaults to `302`.
-
-### `expires` {#cloud-route-partial-expires}
-
-Optional, the duration to cache redirect. Defaults to the `expires` value defined directly under the `redirects` key, but at this level you can fine-tune the expiration of individual partial redirects:
-
-```yaml
-http://{default}/:
-    type: upstream
-    redirects:
-        expires: 1d
-        paths:
-            "/from": { "to": "http://example.com/" }
-            "/here": { "to": "http://example.com/there", "expires": "2w" }
-```
-
-In the preceding example, redirects from `/from` expire in one day, but redirects from `/here` expire in two weeks.
+- **`paths`**—The paths to apply the redirects to.
```

Comment:
I understand the relationship here with Path key-value pairs, but placing them in a bulleted list makes them less easy to find with the navigation. All other configuration topics, such as Applications, variable topics, Caching, etc, use the H3 heading type for their property definitions.

Reply:
I restructured this section to simplify presentation of property descriptions, and provided examples in separate sections.